.travis.yml

```diff
@@ -75,15 +73,15 @@ jobs:
       addons:
         apt:
           packages:
+            - ninja-build
             - binutils-mingw-w64-x86-64
-    - os: linux
-      env: COMPILER_NAME=clang CXX=clang++ CC=clang ASAN=1 ASAN_OPTIONS=detect_odr_violation=0 LSAN_OPTIONS=detect_leaks=0 CFLAGS="-DRZ_ASSERT_STDOUT=1"
 
 cache:
   - ccache
 
 script:
   - export PR_NAME=$(echo $TRAVIS_PULL_REQUEST_SLUG | cut -d'/' -f1)
   - |
+    pip3 install -U --user meson
```

Comment:
i suggest to install ninja from pip as well. Just to be sure we get the latest version.